templates/download/desktop/index.html

```diff
@@ -73,7 +73,59 @@ <h3 class="p-heading--4"><a href="/tutorials/upgrading-ubuntu-desktop">Upgrade U
 
 <section class="p-strip--light">
   <div class="u-fixed-width">
-    <h2>Spin up Ubuntu VMs on Linux, Mac or Windows</h2>
+    <h2>Certified on hundreds of devices</h2>
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <p>Many of the world's biggest PC manufacturers certify their laptops and desktops for Ubuntu, from ultra portable laptops to high end workstations. Ubuntu certified hardware has passed our extensive testing and review process, ensuring that Ubuntu runs well out of the box. In addition, our partners offer select devices preloaded with optimised Ubuntu images.</p>
+      <p><a href="/certified">Read more&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-5 u-vertically-center">
+      <div class="row">
+        <div class="col-small-1 col-medium-1 col-1">
```

Comment:
would it be better to hide the images on mobile and medium screens? 

```suggestion
        <div class="u-hide--small u-hide--medium">
```
<img width="453" alt="image" src="https://user-images.githubusercontent.com/57550290/172154633-6c762170-1bf5-41d2-98d9-49ffc1fa6879.png">